### DISCUSSION TITLE
How to use Git?

### DISCUSSION THREAD

**Discussion by test3211234** [lapce/lapce#2799] [Q&A] [answered]
I don't know how to login and commit and push. It was really easy in VSCode. In Lapce it doesn't do anything and just makes a .git folder.

**Comment by theArianit** [comment]
You should try it with a git app, like gitkraken or so, or even learn git commands and use it from the terminal/powershell.

**Reply by test3211234** [reply]
i know, its just more convenient

**Reply by theArianit** [reply]
yes, I understand you. And I personally think that vsCode made us lazy and dumb :P 
Anyway, be patient. this project isn't even beta yet, I think that feature will come too.

**Comment by panekj** [comment, marked as answer]
Lapce can only initialise repository and add new commits after files have been modified.
